tests/cover/test_type_lookup.py

```diff
@@ -28,7 +28,7 @@
 # Build a set of all types output by core strategies
 blacklist = [
     'builds', 'iterables', 'permutations', 'random_module', 'randoms',
-    'runner', 'sampled_from', 'streaming',
+    'runner', 'sampled_from', 'streaming', 'fspaths',
```

Comment:
I've added the suggested registration now + a test.

I've done the same for pathlib.Path (which only allows str paths) but I haven't used hypothesis with type inference or mypy/annotations so I'm not sure if that is a good idea.